src/main/java/pl/tomaszdziurko/jvm_bloggers/JvmBloggersConfiguration.java

```diff
@@ -33,4 +34,8 @@ public Client getMailingRestClient(@Value("${mailing.apiKey}") String malingApiK
         return client;
     }
 
+    @Bean
+    public CacheManager cacheManager() {
+        return new GuavaCacheManager();
```

Comment:
I am refering to fact that in this case we are not using anything "special" from Guava. SO if this is simple get/put...why not simple ConcurrentMap...KISS :)